src/types.ts

```diff
@@ -75,35 +75,53 @@ type SectionType = 'autocomplete' | 'recommendations' | 'custom';
 
 export type SectionConfiguration = {
   type?: SectionType;
-  identifier: string;
   displayName?: string;
   numResults?: number;
   // This property will only take effect when using the component and not the hook
   displaySearchTermHighlights?: boolean;
   ref?: React.RefObject<HTMLElement>;
 };
 
-export interface AutocompleteSection extends SectionConfiguration {
+export interface AutocompleteSectionConfiguration extends SectionConfiguration {
   type?: 'autocomplete';
-  data: Item[];
+  indexSection: string;
```

Comment:
Are we renaming this to indexSectionName for clarity?

Reply:
Are we?